<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modbus Slave</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/pyModSlaveQt.qrc">
    <normaloff>:/img/Bus.png</normaloff>:/img/Bus.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblSettings">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayoutSettings">
      <item row="0" column="3">
       <widget class="QSpinBox" name="sbSlaveID">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblModbusMode">
        <property name="text">
         <string>Modbus Mode</string>
        </property>
        <property name="buddy">
         <cstring>cmbModbusMode</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbModbusMode">
        <item>
         <property name="text">
          <string>RTU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblSlaveID">
        <property name="text">
         <string>Slave ID</string>
        </property>
        <property name="buddy">
         <cstring>sbSlaveID</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblCmds">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Commands</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayoutCmds">
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btStartStop">
        <property name="toolTip">
         <string>Start-Stop</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="icons/pyModSlaveQt.qrc">
          <normaloff>:/img/run-16.png</normaloff>
          <normalon>:/img/stop-16.png</normalon>:/img/run-16.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spInterval">
        <property name="minimum">
         <number>500</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>500</number>
        </property>
        <property name="value">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblSimCycle">
        <property name="text">
         <string>Simulate Every (msec)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSerial_RTU"/>
    <addaction name="actionTCP"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionIO_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/close16.png</normaloff>:/img/close16.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSerial_RTU">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/edit-16.png</normaloff>:/img/edit-16.png</iconset>
   </property>
   <property name="text">
    <string>Modbus RTU...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTCP">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/network-16.png</normaloff>:/img/network-16.png</iconset>
   </property>
   <property name="text">
    <string>Modbus TCP...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/info16.png</normaloff>:/img/info16.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBus_Monitor">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/view-16.png</normaloff>:/img/view-16.png</iconset>
   </property>
   <property name="text">
    <string>Bus Monitor</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/options-16.png</normaloff>:/img/options-16.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCoils">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/DO.png</normaloff>:/img/DO.png</iconset>
   </property>
   <property name="text">
    <string>Coils</string>
   </property>
  </action>
  <action name="actionDigital_Inputs">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/DI.png</normaloff>:/img/DI.png</iconset>
   </property>
   <property name="text">
    <string>Digital Inputs</string>
   </property>
  </action>
  <action name="actionHolding_Registers">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/AO.png</normaloff>:/img/AO.png</iconset>
   </property>
   <property name="text">
    <string>Holding Registers</string>
   </property>
  </action>
  <action name="actionAnalog_Inputs">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/AI.png</normaloff>:/img/AI.png</iconset>
   </property>
   <property name="text">
    <string>Analog Inputs</string>
   </property>
  </action>
  <action name="actionIO_Data">
   <property name="icon">
    <iconset resource="icons/pyModSlaveQt.qrc">
     <normaloff>:/img/IO.png</normaloff>:/img/IO.png</iconset>
   </property>
   <property name="text">
    <string>IO Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spInterval</tabstop>
 </tabstops>
 <resources>
  <include location="icons/pyModSlaveQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
